tidb-monitoring-api.md

```diff
@@ -0,0 +1,82 @@
+---
+title: TiDB Monitoring API
+summary: Learn the API of TiDB monitoring services.
+category: how-to
+---
+
+# TiDB Monitoring API
+
+You can use the following two types of interfaces to monitor the TiDB cluster state:
+
+- [The state interface](#use-the-state-interface): this interface uses the HTTP interface to get the component information.
+- [The metrics interface](#use-the-metrics-interface): this interface uses Prometheus to record the detailed information of the various operations in components and views these metrics using Grafana.
+
+## Use the state interface
+
+The state interface monitors the basic information of a specific component in the TiDB cluster. It can also act as the monitor interface for Keepalive messages. In addition, the state interface for the Placement Driver (PD) can get the details of the entire TiKV cluster.
+
+### TiDB server
+
+- TiDB API address: `http://${host}:${port}`
+- Default port: `10080`
+- Details about API names: see [TiDB HTTP API](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)
```

Comment:
Could you provide the source of this line? Because I didn't see it on [the Chinese `dev` version](https://pingcap.com/docs-cn/dev/tidb-monitoring-api/#tidb-server).